pages/index.vue

```diff
@@ -28,14 +28,19 @@
     <section id="register">
       <!-- TODO: Create the registration section -->
     </section>
+    <section id="footer">
```

Comment:
te rog sa adaugi Footer-ul in `layouts/default.vue`, pentru ca e o componenta "globala". also, acolo sugeram sa ii spunem "`TheFooter`", ca sa nu ne lovim de [problema asta](https://vuejs.org/v2/style-guide/#Multi-word-component-names-essential). Ideea de a prefixa cu "the" vine tot din [styleguide, de aici](https://vuejs.org/v2/style-guide/#Single-instance-component-names-strongly-recommended)